README.md

```diff
@@ -35,6 +35,10 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 You must sign a [Contributor License Agreement](https://cla.microsoft.com/) before submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to submit a request via the [form](https://cla.microsoft.com/) and then electronically sign the CLA when you receive the email containing the link to the document. You need to sign the CLA only once to cover submission to any Microsoft OSS project. 
 
+### 2.3 Code Formatting
+
+All Objective-C files follow LLVM coding style (with a few exceptions) and are formatted accordingly. If you use Xcode for your contribution, use our [.clang_format](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/.clang-format) file to make sure files have the correct code format before submitting PRs.
```

Comment:
We are assuming that people know how to use a `.clang_format` file. Can we put in more info on how to get a formatting tool and how to use it?

Reply:
I tried to put a link with more details for tools but didn't do it because of license issue + unknown side effects. Really wonder it is safe to link to a tool and give descriptions here in legal perspective. Let's take offline and talk about it.

Reply:
Maybe it is okay since we are not distributing any binaries and just put a link with descriptions but I want to make sure we don't have any concerns on our official doc.

Reply:
This should be fine since git clone is not pulling the tool.